File: abc/abc203/a.cs
// https://atcoder.jp/contests/abc203/submissions/46503095

using System;

public class A
{
  static void Main()
  {
    string[] t = Console.ReadLine().Split(" ");

    for(int i = 0; i < t.Length; i++)
    {
      for(int j = i + 1; j < t.Length; j++)
      {
        if(int.Parse(t[i]) == int.Parse(t[j]))
        {
          Console.WriteLine(t[3 - i - j]);
          return;
        }
      }
    }

    Console.WriteLine(0);
  }
}

File: abc/abc203/a.py
# https://atcoder.jp/contests/abc203/submissions/34672195

x = list(map(int, input().split()))
x.sort()
if x[0] == x[1]: print(x[2])
elif x[1] == x[2]: print(x[0])
else: print(0)
